core/src/main/java/io/confluent/kafka/schemaregistry/rest/SchemaRegistryConfig.java

```diff
@@ -720,10 +733,34 @@ public String interInstanceProtocol() {
     return getString(INTER_INSTANCE_PROTOCOL_CONFIG);
   }
 
+  /**
+   * Gets the inter.instance.listener.name setting.
+   */
+  public String interInstanceListenerName() {
+    return getString(INTER_INSTANCE_LISTENER_NAME_CONFIG);
+  }
+
   public List<String> whitelistHeaders() {
     return getList(INTER_INSTANCE_HEADERS_WHITELIST_CONFIG);
   }
 
+  public Map<String, Object> getOverriddenSslConfigs(NamedURI listener) {
```

Comment:
Yeah, I considered that - but `getSslConfig` is private. Also, even if it was public, it returns a `SslConfig` object that is not readily usable in SslFactory. Would have to either add a new method in rest-utils that returns what we want or make a lot of changes in `SslFactory`. Hence, wrote this code that does something very similar.

Reply:
@rigelbm , do you have an opinion on this?

Reply:
What is SslFactory (do you have a link?) and can it be made to work with SslConfig? If so, I would rather make `getSslConfig` `protected final` in rest-utils.

Reply:
The reason I created SslConfig was to avoid applications accessing these configurations by name. Even in rest-utils itself I found a wrong-name usage bug. If you have avoid using a Map, and use SslConfig instead, that would be ideal.

Reply:
@rigelbm This is the SslFactory that we use. 
https://github.com/confluentinc/schema-registry/blob/master/client/src/main/java/io/confluent/kafka/schemaregistry/client/security/SslFactory.java

Reply:
I do agree it would be cleaner if SslFactory.java uses SslConfig instead of the Map. 
@rayokota Do you think it makes sense?

Reply:
I just noticed that SslFactory, which is in the schema-registry client package, has no dependency on rest-utils - so this change would involve adding this new dependency in client code as well.

Reply:
Let's not have the SR client depend on rest-utils, as that could bring in a lot of transitive dependencies.

Reply:
@sanjay-awatramani I discussed this with @rayokota and his suggestion is to not add a client dependency on rest-utils, as that could bring in a lot of transitive dependencies. I will keep the implementation as it is for now.

Reply:
Agree with this